tests/expectations/tests/issue-537-repr-packed-n.rs

```diff
@@ -40,16 +40,19 @@ fn bindgen_test_layout_AlignedToOne() {
     );
 }
 /// This should be be packed because Rust 1.33 has `#[repr(packed(N))]`.
-#[repr(C, packed(2))]
+#[repr(C, packed)]
 #[derive(Debug, Default, Copy, Clone)]
-pub struct AlignedToTwo {
+pub struct AlignedToTwo__packed {
     pub i: ::std::os::raw::c_int,
 }
+#[derive(Debug, Default, Copy, Clone)]
+#[repr(C, align(2))]
+pub struct AlignedToTwo(AlignedToTwo__packed);
```

Comment:
This should be public, how do folks access the fields otherwise? More to the point, we should try to only add this wrapper when absolutely needed (I think `packed && max_field_align < explicit_align`, right?), and also we should probably `impl Deref` and `DerefMut` on the struct to get more convenient access / avoid most code breakage as a result of this change.

Reply:
About Deref, Is there another wrapper mechanism to do it? Or this is the first place to add those Deref stuff?
The condition is logically write but I have no idea how to do yet.

Reply:
Counter example about the condition.
```
struct B {
  uint8_t a;
  uint16_t b;
  uint8_t c;
} __attribute__((aligned(2), packed));
```
This is packed but not `max_field_align < explicit_align` (2 < 2).